pkg/listener/kafka/kafka-consumer.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	brokers := os.Getenv("BROKER_LIST")
+	if brokers == "" {
+		logrus.Fatalf("broker can't be empty. Please set env BROKER_LIST.")
+	}
+	brokerList := strings.Split(brokers, ",")
+	topic := os.Getenv("TOPIC")
```

Comment:
I would assume that there may be more than one topic

Reply:
sure it should be multiple topics and partitions I guess. Adding follow-up commit for that